lib/command/setup_app.rb

```
@@ -24,6 +27,15 @@ def call
       end
 
       Cpl::Cli.start(["apply-template", *templates, "-a", config.app])
+
+      return if config.options[:skip_secret_access_binding] ||
+                cp.fetch_identity(app_identity).nil? ||
```

would rather have an error and not silently skip